grunt_tasks/plugin.js

```diff
@@ -150,26 +191,86 @@ module.exports = function (grunt) {
                     }
                 }
             });
+
+            // If the plugin config has no webpack section, no defaultLoaders
+            // property in the webpack section, or the defaultLoaders is
+            // explicitly set to anything besides false, then augment the
+            // webpack loader configurations with the plugin source directory.
+            if (!config.webpack || config.webpack.defaultLoaders === undefined || config.webpack.defaultLoaders !== false) {
+                console.log('using!');
+                var numLoaders = grunt.config.get('webpack.options.module.loaders').length;
+                for (var i = 0; i < numLoaders; i++) {
+                    var selector = 'webpack.options.module.loaders.' + i + '.include';
+                    var loaders = grunt.config.get(selector);
+                    grunt.config.set(selector, loaders.concat([path.resolve(dir)]));
+                }
+            }
+
+            var newConfig = webpackHelper(grunt.config.get('webpack.options'), helperConfig);
+            grunt.config.set('webpack.options', newConfig);
         }
 
+        grunt.registerTask('npm-install', 'Install plugin NPM dependencies', function (plugin) {
+            var done = this.async();
+
+            var target = path.resolve('node_modules_' + plugin);
+
+            if (fs.existsSync(target + '/node_modules')) {
```

Comment:
So really I think there should just be another NPM script to install plugin dependencies somehow, rather than this more convoluted approach. i.e. I'd like to be able to say something like `npm run install:plugin -- --plugins=foo` and have it do what my Grunt task does.

I went with this approach instead because it's status quo but I planned to address this updatability issue somehow in a separate PR after this one was merged.